recipes/asl/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "asl" %}
+{% set version = "3.1.0" %}
+{% set sha256 = "587c1a88f4c8f57bef95b58a8586956145417c8039f59b1758365ccc5a309ae9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/ampl/mp/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - cmake
+
+test:
+  files:
+    - miniampl
+
+about:
+  home: http://ampl.github.io
+  license: custom
```

Comment:
Looks kind of like [HPND]( https://opensource.org/licenses/HPND ) maybe. Please double check. Also IANAL